must-call-checker/src/main/java/org/checkerframework/checker/mustcall/NotOwning.astub

```diff
@@ -41,13 +41,13 @@ class SocketChannel extends AbstractSelectableChannel implements ByteChannel, Sc
 
 class ServerSocketChannel extends AbstractSelectableChannel implements NetworkChannel {
     protected ServerSocketChannel(SelectorProvider arg0);
-    static ServerSocketChannel open() throws IOException;
+    @MustCallChoice static ServerSocketChannel open(@MustCallChoice ServerSocketChannel this) throws IOException;
     int validOps();
-    ServerSocketChannel bind(SocketAddress arg0) throws IOException;
-    ServerSocketChannel bind(SocketAddress arg0, int arg1) throws IOException;
+    @MustCallChoice ServerSocketChannel bind(@MustCallChoice ServerSocketChannel this, SocketAddress arg0) throws IOException;
+    @MustCallChoice ServerSocketChannel bind(@MustCallChoice ServerSocketChannel this, SocketAddress arg0, int arg1) throws IOException;
```

Comment:
I think these are correct - the Javadoc for `bind` says that it returns the same channel it's called on.